rcsbapi/data/data_query.py

```diff
@@ -154,24 +158,27 @@ def get_editor_link(self) -> str:
         editor_base_link = str(const.DATA_API_ENDPOINT) + "/index.html?query="
         return editor_base_link + urllib.parse.quote(self._query)
 
-    def exec(self) -> Dict[str, Any]:
+    def exec(self, progress_bar: bool = False) -> Dict[str, Any]:
         """POST a GraphQL query and get response
 
         Returns:
             Dict[str, Any]: JSON object
         """
-        batch_size = 50
+        batch_size = 5000
```

Comment:
I'd recommend just moving `batch_size` to be another argument to `exec(...)` with default 5000. That way, the user can easily override it when they run it on their own if they wish.